proxy/traefik/config.yml

```diff
@@ -0,0 +1,35 @@
+http:
```

Comment:
Can you add a README file to this directory to explain what it’s about? It would also be helpful to mention the `FRONTEND_URL` env var